src/modules/lesson/entities/lesson.entity.ts

```diff
@@ -0,0 +1,52 @@
+import { ApiProperty, IntersectionType } from '@nestjs/swagger';
+import { getValueByEnum } from '@src/common/common';
+import { DateResponseType } from '@src/types/date-response.type';
+import { IdResponseType } from '@src/types/id-response-type';
+import { Lesson as LessonModel } from '@prisma/client';
+import { LessonLevelId } from '@src/constants/enum';
+
+export class LessonEntity
+  extends IntersectionType(IdResponseType, DateResponseType)
+  implements LessonModel
+{
+  @ApiProperty({
+    example: 1,
+    description: '출제자가 생각한 과제의 난이도 {상 : 1, 중 : 2, 하 : 3}',
+    required: true,
+    type: 'number',
+    enum: getValueByEnum(LessonLevelId, 'number'),
+  })
+  levelId: number;
```

Comment:
```suggestion
  levelId: LessonLevelId;
```
일반 상수가 아닌 Enum을 사용하는게 타입으로 쓸 수 있다는 점도 있을거같은데 적극 사용하면 좀 더 엄격하게 타입을 관리할 수 있을거같네여

Reply:
enum 적극 사용하는 것을 습관화 하도록 하겠습니다!